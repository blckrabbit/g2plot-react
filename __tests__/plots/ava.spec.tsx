import React from 'react'
import ReactDOM from 'react-dom'
import { create } from 'react-test-renderer'
import AVAChart from '../../src/plots/ava'

describe('AVAChart', () => {
  test('should render without crashed', () => {
    const div = document.createElement('div')
    ReactDOM.render(<AVAChart data={[]} />, div)

    ReactDOM.render(<AVAChart data={[{ f1: 'a', f2: 100 }]} />, div)
  })

  test('match snapshot', () => {
    const renderer = create(<AVAChart data={[]} />)

    expect(renderer.toJSON()).toMatchSnapshot()
  })
})
