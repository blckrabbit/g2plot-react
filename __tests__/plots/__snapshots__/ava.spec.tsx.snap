// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AVAChart match snapshot 1`] = `<div />`;
